ml-proto/test/left-to-right.wast

```diff
@@ -1,20 +1,22 @@
 (module
-  (memory 1)
+  (memory 9)
   
   ;; The idea is: We reset the memory, then the arithmetic instruction calls $*_left and $*_right.
   ;; $*_left stores the number 1, and $*_right stores the number 2 (both at address 0)
   ;; Then we read the value at address 0. If it's 0, the VM did an unwanted optimzation.
```

Comment:
This comment is now stale, since the loads and stores below use address 8. Also, is there a reason for switching from 0 to 8?